src/components/basic/SearchInput/SearchInput.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from 'react'
```

Comment:
please add license headers to all newly created files

Reply:
okay sure